indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -228,7 +245,7 @@
                       'merge': 'visited',
                       'mark_as_duplicate': 'strong',
                       'change_tracks': 'warning'} %}
-    <span class="i-tag outline {{ mapping[review.proposed_action.name] }}">
+    <span class="bold underline semantic-text {{ mapping[review.proposed_action.name] }}">
```

Comment:
I was about to go for that initially, but is it worth adding the class just for `bold` and `underline`? We need to add `.semantic-text` anyway to apply the color variations from the mapping.